client/index.js

```diff
@@ -35,7 +36,11 @@ let lastAppProps
 export const router = createRouter(pathname, query, getURL(), {
   Component,
   ErrorComponent,
-  err
+  err,
+  formatURL: exported && function (buildId, route) {
+    route = route && route.replace(/\/$/, '')
+    return route + '/index.json'
```

Comment:
Probably better rewritten as

```js
return route
  ? route.replace(/\/$/, '') + '/index.json'
  : '/index.json'
```

(also doesn't reassign the function argument)